website/static/js/institutionProjectSettings.js

```diff
@@ -20,10 +20,8 @@ var ViewModel = function(data) {
             url,
             {isCors: true}
         ).done(function (response) {
-            if (response.data.embeds.institutions.data.length){
-                self.availableInstitutions(response.data.embeds.institutions.data);
-                self.loading(false);
-            }
+            self.availableInstitutions(response.data.embeds.institutions.data.length ? response.data.embeds.institutions.data: []);
+            self.loading(false);
```

Comment:
Ah I see. Thanks for the clarification. Yes, there should be a an error message displayed to the user if this fails. Can you add that to this PR?